2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_090034-qmsyp7hr/logs/debug.log
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_090034-qmsyp7hr/logs/debug-internal.log
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-28 09:00:34,415 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: swiui98q
2024-10-28 09:00:34,417 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/swiui98q
2024-10-28 09:00:34,417 INFO    MainThread:405 [jupyter.py:save_history():459] not saving jupyter history
2024-10-28 09:00:34,417 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:00:34,417 INFO    MainThread:405 [wandb_init.py:_jupyter_teardown():462] cleaning up jupyter logic
2024-10-28 09:00:34,417 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:00:34,417 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:00:34,418 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:00:37,413 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:00:37,413 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:00:37,417 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:00:37,421 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:00:37,421 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:00:37,422 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:00:37,422 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:00:37,429 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:00:37,438 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:00:37,438 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:00:37,932 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:00:38,102 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:00:38,102 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:00:38,102 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:00:38,102 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:00:38,104 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:02:04,415 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_090204-ybnscgic/logs/debug.log
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_090204-ybnscgic/logs/debug-internal.log
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_init.py:_jupyter_setup():480] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f28bdcdfc70>
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:02:04,416 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-28 09:02:04,417 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: qmsyp7hr
2024-10-28 09:02:04,418 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/qmsyp7hr
2024-10-28 09:02:04,418 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:02:04,418 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:02:04,418 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:02:06,825 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:02:06,826 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:02:06,829 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
